src/components/Calculator.js

```diff
@@ -25,41 +24,47 @@ const Calculator = () => {
   };
 
   return (
-    <div className="calculator">
-      <div className="display">
-        {calculatorData.next !== null
-          ? calculatorData.next
-          : calculatorData.total || '0'}
+    <div className="calculator-container">
+      <div className="calculator-text">
+        <h1>Math Magicians</h1>
+        <h2>Let&apos;s do some math</h2>
```

Comment:
Nice job so far😉😉😉 everything looks good👍🏼. but let's try to make the nav bar section match the layout of the design😉. take note of the `math magician` header and the nav link and kindly ensure they are positioned in the right places as shown in the design. also, try to ensure your navbar occupies the full width of the page as shown in the design👍🏼.

design
<img width="899" alt="image" src="https://github.com/Kaiserabbas/math-magician/assets/64643995/c98d1cf0-48a1-4d2d-8547-11d2cc910fd0">

yours
<img width="1440" alt="image" src="https://github.com/Kaiserabbas/math-magician/assets/64643995/9279f5aa-e953-440b-bb9d-e1781e5ff665">

The logo should be placed on the nav bar and the navbar should take a 100% width to match the layout of the wireframe👍🏼.

Reply:
Thank you for for your kind review, I have addressed all the issues raised.